### PR TITLE
[nnpackage] Add NNPackage example

### PR DESCRIPTION
**Pull request by hseok-oh**
- Readme documentation
- Show version by directory name
- Add nnpackage for validation test

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
It's not nice solution. But by this PR, we can run `nnfw_api_gtest` without model download.

FYI, example directory size: 196K
```
8.0K    ./v1.0.0/add/metadata
16K     ./v1.0.0/add
28K     ./v1.0.0/while_dynamic/metadata/tc
36K     ./v1.0.0/while_dynamic/metadata
52K     ./v1.0.0/while_dynamic
8.0K    ./v1.0.0/add_invalid_manifest/metadata
16K     ./v1.0.0/add_invalid_manifest
28K     ./v1.0.0/if_dynamic/metadata/tc
36K     ./v1.0.0/if_dynamic/metadata
76K     ./v1.0.0/if_dynamic
164K    ./v1.0.0
12K     ./v1.1.0/one_op_in_tflite/metadata
20K     ./v1.1.0/one_op_in_tflite
24K     ./v1.1.0
196K    .
```

File: nnpackage/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Model file: TensorFlow Lite model
+- Only one `ADD` operation
+
+### add_invalid_manifest
+
+- Model file: TensorFlow Lite model
+- Only one `ADD` operation
+- Invalid menifest: invalid json format
+
+### if_dynmic
```
Comment (wateret) [inline nnpackage/examples/README.md:23]:
```suggestion
### if_dynamic
```

File: nnpackage/examples/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Model file: TensorFlow Lite model
+- `IF` operation example with input and output example
+
+### while_dynmic
```
Comment (wateret) [inline nnpackage/examples/README.md:28]:
```suggestion
### while_dynamic
```

**Comment by wateret** [comment]
+1 since having those binary files in the repo, since they are small.

**Comment by hseok-oh** [comment]
I've fixed typo. PTAL

**Review by glistening — APPROVED** [review]
LGTM

**Review by wateret — APPROVED** [review]
LGTM

**Pull request merged**